.github/kokoro/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+SCRIPT_SRC="$(realpath ${BASH_SOURCE[0]})"
+SCRIPT_DIR="$(dirname "${SCRIPT_SRC}")"
+INSTALL_DIR="$(pwd)/install"
+
+export CMAKE_FLAGS="-GNinja -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}"
+export BUILD_TOOL=ninja
+source ${SCRIPT_DIR}/common.sh
+
+echo
+echo "========================================"
+echo "Running install tests (make install)"
+echo "----------------------------------------"
+(
+	pushd build
+	export VPR_NUM_WORKERS=${CORES}
+	ninja -j${MAX_CORES} install
+	popd
+)
+echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Compressing and uploading install dir"
+echo "----------------------------------------"
+(
+	tar -c --use-compress-program="pigz" -f install-${GIT_DESCRIBE}.tar.gz ${INSTALL_DIR}
```

Comment:
The output binary should probably have symbiflow-arch-defs or symbiflow in the name somewhere?